File: sdkwork-app-sdk-csharp/Models/MusicSimilarForm.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class MusicSimilarForm
    {
        public string? ReferenceUrl { get; set; }
        public int? Duration { get; set; }
        public string? Model { get; set; }
    }
}

File: sdkwork-app-sdk-java/src/main/java/com/sdkwork/app/model/MusicSimilarForm.java
package com.sdkwork.app.model;

public class MusicSimilarForm {
    private String referenceUrl;
    private Integer duration;
    private String model;

    public String getReferenceUrl() {
        return this.referenceUrl;
    }
    
    public void setReferenceUrl(String referenceUrl) {
        this.referenceUrl = referenceUrl;
    }

    public Integer getDuration() {
        return this.duration;
    }
    
    public void setDuration(Integer duration) {
        this.duration = duration;
    }

    public String getModel() {
        return this.model;
    }
    
    public void setModel(String model) {
        this.model = model;
    }
}
